docs/handbook/common-tasks.md

```diff
@@ -45,6 +45,87 @@ Print versions are also available.
 
 ## Basic system administration
 
+### Basic system information
+
+#### System processes
+
+```markdown
+prstat
+```
+
+This command provides a host of information on all processes running on the system.
+Some of the information provided is as follows:
+
+- percentage of CPU used by each process
+- amount of memory consumed by each process
+- unique id of each process (which can, for example, be used to stop the process)
+
+#### Disk usage
+
+```markdown
+df -h
+```
+
+Provides information on disk size, amount of space used and available free space
+for all attached storage devices. The `-h` option reports this information in
+human readable format.
+
+#### Largest files in a directory
+
+Go to the directory using the `cd` command and issue the following command:
+
+```markdown
+du | sort -n
+```
+
+This will list the size of each file in the current directory and all
+subdirectories, starting with the smallest up to the largest files.
+
+#### Who is logged on to the system
+
+```markdown
+listusers
+```
+
+#### List all software packages installed on the system
+
+```markdown
+pkginfo
+```
+
+### System shutdown, reboot, ...
+
+OpenIndiana defines a number of different system states known as run-levels. You
+can change from one system state to another by using the `shutdown` command and
+specify the run-level using the `i` option. You can always determine the
+run-level via `who -r`.
+
+You must be root or have root privilages (e.g., using `sudo`) to send the system
```

Comment:
Good idea to reference the man pages (will also require a _brief_ section on man pages).  Did not want to go into any detail because I wanted this stuff to be short, so you're always cherry picking.  But it is really a natural continuation, for more details refer to the man pages, thanks.

The pdf script started out as a private thing a day or so ago, by the evening I realised this might be useful to others, so I cleaned it up a bit, added a help etc; but the quality leaves much to be desired and more work is required to fine tune. But it is something I can use _now_. If we've got a better way of doing this on the build server, then I shall not wast any more time in the script. I also hate that the pdfs are in the same dir as the *md files, so it would be easy to modify the script to put all the pdfs into a pdf dir.  All in agreement?
I can and shall relicence under CDDL